packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -1065,18 +1103,22 @@ contract Deploy is Deployer {
                     _blockOracle: BlockOracle(mustGetAddress("BlockOracle"))
                 })
             );
-
-            uint8 rawGameType = GameType.unwrap(_gameType);
-            console.log(
-                "DisputeGameFactoryProxy: set `FaultDisputeGame` implementation (Backend: %s | GameType: %s)",
-                rawGameType == 0 ? "Cannon" : "Alphabet",
-                vm.toString(rawGameType)
-            );
+        }
+        uint8 rawGameType = GameType.unwrap(_gameType);
+        string memory gameTypeString;
+        if (rawGameType == 0) {
+            gameTypeString = "Cannon";
+        } else if (rawGameType == 254) {
+            gameTypeString = "OutputBisectionAlphabet";
+        } else if (rawGameType == 255) {
+            gameTypeString = "Alphabet";
         } else {
-            console.log(
-                "[WARN] DisputeGameFactoryProxy: `FaultDisputeGame` implementation already set for game type: %s",
-                vm.toString(GameType.unwrap(_gameType))
-            );
+            gameTypeString = "Unknown";
         }
+        console.log(
+            "DisputeGameFactoryProxy: set `FaultDisputeGame` implementation (Backend: %s | GameType: %s)",
```

Comment:
tiny nit: template `FaultDisputeGame` and switch based on the game type in this log.

Reply:
er wait, nvm, I think `GameType` is sufficient here. can ignore.